docs/contributing/setup.md

```diff
@@ -10,12 +10,14 @@ You will need to install these tools on your machine:
  - Python 2.7 - [Windows](https://www.python.org/downloads/windows/), [macOS](https://www.python.org/downloads/mac-osx/)
     - **Windows developers**: *Let Python install into the default suggested path
       (`c:\Python27`), otherwise you'll have to configure node-gyp manually with
-      the path which is annoying.*
+      the path which is annoying. You will also need to make sure the **Add python.exe to Path** option is selected.*
  - **macOS:** Xcode and Xcode Command Line Tools (Xcode -> Preferences -> Downloads)
  - **Windows:** Visual Studio 2015 or [Visual C++ Build Tools](http://go.microsoft.com/fwlink/?LinkId=691126)
-    - Run `npm config set msvs_version 2015` after installing the build tools.
+    - If Visual Studio is already installed you will need to make sure you have Visual C++ installed as part of it. The **Common Tools for Visual C++ 2015** feature is all that's required.
+    - Run `npm config set msvs_version 2015` after installing Visual C++ or the build tools. You may need to do this even if they're already installed.
```

Comment:
Given this is something it looks like we have to do always, let's call it out after all these dependencies, and before the "With these things installed, open a shell and validate you have these commands" paragraph.